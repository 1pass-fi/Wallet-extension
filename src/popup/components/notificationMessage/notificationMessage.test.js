import '@babel/polyfill'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import NotificationMessage from '.'

describe('Test for NotificationMessage component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const { container } = render(<NotificationMessage />)
      expect(container).toMatchSnapshot()
    })
  })
})
