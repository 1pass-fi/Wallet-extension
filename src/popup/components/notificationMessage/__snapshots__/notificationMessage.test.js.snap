// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for NotificationMessage component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="notification-message "
  />
</div>
`;
